src/application/clients/crux-gateway-bridge-without-auth.ts

```diff
@@ -21,7 +21,7 @@ tcpServer.on("connection", (socket) => {
     // upgrade the tcp socket into a strong-pubsub-connection
     const bridge = new Proxy(
       new MqttConnection(socket),
-      new Client({host: config.HOST_URL.BROKER_HOST, port: config.PORTS.BROKER_PORT}, Adapter),
+      new Client({host: process.env.BROKER_HOST, port: config.PORTS.BROKER_PORT}, Adapter),
```

Comment:
Add fallbacks for process.env.BROKER_HOST to config.HOST_URL.BROKER_HOST

also add process.env.BROKER_PORT

Reply:
Done